pyproject.toml

```diff
@@ -20,6 +20,7 @@ keywords = ["malloy"]
 dependencies = [
     "duckdb>=0.8.0, <1.0.0",
     "google-cloud-bigquery",
+    "snowflake-connector-python",
```

Comment:
You may need to specify a version here, for some reason `requirements.dev.txt` and `requirements.ipyton.txt` have 2.6.1, while `requirements.txt` has 3.6.0. 2.6.1 is so old it pre-dates `connections.toml`, which meant the tests weren't even connecting.

Reply:
good catch. updating it.

Reply:
tests did run but snowflake connections might have failed
we probably need
```
$ cat ~/.snowflake/connections.toml
[default]
account = "account_here"
user = "username_here"
password = "password_here"
warehouse = "warehouse_here"
database = "MALLOYTEST"
schema = "MALLOYTEST"
```
in the container